operator/pkg/reconciliation/reconcile_racks.go

```diff
@@ -264,9 +264,15 @@ func (rc *ReconciliationContext) CheckRackPodTemplate() result.ReconcileResult {
 				return result.Error(err)
 			}
 
+			if err := rc.enableQuietPeriod(20); err != nil {
```

Comment:
Is there any reason to not just pause here for 20 seconds, as opposed to doing nothing on subsequent reconcile calls for 20 seconds?

Reply:
Yes - for #174 we want one Cass Operator to handle more CassDCs gracefully, so blocking in the middle of the reconcile loop for any of them is not great.

Reply:
That makes sense. :+1: